src/persistents_with_classid.cc

```diff
@@ -113,28 +112,30 @@ class PersistentHandleWithClassIdVisitor : public PersistentHandleVisitor {
     }
 
     void report_(Isolate* isolate, Local<Function> fn, PersistentHandleWithClassId* handle) {
+      Local<v8::Context> context = isolate->GetCurrentContext();
       Local<Value> argv[] = { Integer::New(isolate, handle->id()), handle->value() };
-      fn->Call(Null(isolate), 2, argv);
+      fn->Call(context, Null(isolate), 2, argv);
```

Comment:
Looks like this change would break earlier versions of v8?
That'd be fine if we use older versions for agents of older N|Solids.